app/templates/partials/question.html

```diff
@@ -2,13 +2,14 @@
 
 {% set invalid = question.is_valid == False %}
 
+{% set question_number = question.schema_item.number %}
 {% set question_title = question.schema_item.title %}
 {% set question_subtitle = question.schema_item.subtitle %}
 {% set question_guidance = "" %}
 
 {% set question_titles %}
   {%- if question_title -%}
-    <h2 class="question__title neptune">{{question_title}}</h2>
+<h2 class="question__title neptune"><span>{{question_number}}. </span>{{question_title}}</h2>
```

Comment:
Question number is optional, so we shouldn't output the span if it's not in the schema.